src/assets/theme/buttons.tsx

```diff
@@ -0,0 +1,71 @@
+import { COLOR_MAP } from 'common/colors';
+import fonts from 'common/theme/fonts';
+
+export enum ButtonType {
+  FILL,
+  OUTLINE,
+  TEXT,
+}
+
+export interface ButtonsTheme {
+  background: string;
+  border: string;
+  text: string;
+  backgroundHover: string;
+  borderHover: string;
+  textHover: string;
+  disabledBackground: string;
+  disabledText: string;
+  icon: string;
+  disabledIcon: string;
+  fontFamily: any; // fix
+}
+
+const newBlue = '#3567FD';
+const newBlueDark = '#002CB4';
+const newBlueLight = 'rgba(53, 103, 253, .15)';
+const lightGrayFigma = '#e0e0e0';
```

Comment:
will move all of these into colors or palette in a separate pr, probably when implementing the color map josh linked in figma